pom.xml

```diff
@@ -296,9 +296,16 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-enforcer-plugin</artifactId>
         <version>3.0.0-M1</version>
+        <dependencies>
+          <dependency>
+            <groupId>com.google.cloud.tools</groupId>
+            <artifactId>linkage-checker-enforcer-rules</artifactId>
+            <version>0.1.1</version>
+          </dependency>
+        </dependencies>
         <executions>
           <execution>
-            <id>enforce</id>
+            <id>convergence</id>
```

Comment:
Is this execution necessarily only `convergence`?

Reply:
This is just a name. It doesn't really mean anything, but they're can't be two rules named "enforce" or Maven complains.

Reply:
Right, but I don't know what the default behavior is. Shouldn't this remain just `enforce`. Why did we change it?

Reply:
It should never have been enforce. That's just copy-pasta from the docs.